tools/templates/moar/Makefile.in

```diff
@@ -106,7 +106,7 @@ m-install: m-all
 	$(CP) $(NQP_MOAR) @q($(DESTDIR)$(NQP_LIB_DIR))@
 	$(CP) $(P5QREGEX_MOAR) @q($(DESTDIR)$(NQP_LIB_DIR))@
 	$(CP) @nfp(src/vm/moar/profiler/template.html)@ @nfpq($(DESTDIR)$(NQP_LIB_DIR)/profiler/template.html)@
-	$(PERL5) @shquot(@script(install-moar-runner.pl)@)@ "$(DESTDIR)" @q($(PREFIX))@ @q($(NQP_LIB_DIR))@ @nfpq(@moar@)@
+	$(PERL5) @shquot(@script(install-moar-runner.pl)@)@ "$(DESTDIR)" @q($(PREFIX))@ @q($(NQP_LIB_DIR))@ @nfpq(@moar@)@ @relocatable@
```

Comment:
According to https://github.com/perl6/nqp-configure/pull/6/commits/b8e9268be09d70bda7902d1e85a22d99ce64ca12 `relocatable` variable wouldn't be set.

Reply:
Good catch! Fixing.

Reply:
Not really. I have found you setting this variable in `NQP::Config::Rakudo`. So, perhaps you need it anyway?

Reply:
I moved the `configure_relocatability` function back to nqp-configure as you proposed. So this is fixed.